### PR TITLE
[XM] Missing NSUserNotification bindings

### PR DESCRIPTION
**Pull request by chamons**
- https://bugzilla.xamarin.com/show_bug.cgi?id=38118

### CODE REVIEW

**Comment by spouliot** [comment]
introspection-mac all (3) failed

File: src/foundation.cs [hunk 1]
```diff
@@ -12577,6 +12577,49 @@ public interface NSUserNotification : NSCoding, NSCopying {
...
+		NSUserNotificationAction[] AdditionalActions { get; set; }
+
+		[Mac (10, 10)]
+		[NullAllowed, Export ("additionalActivationAction", ArgumentSemantic.Copy)]
+		NSUserNotificationAction AdditionalActivationAction { get; }
+	}
+
+	[Mac (10,10)]
+	[BaseType (typeof(NSObject))]
+	public interface NSUserNotificationAction : NSCopying, INSCopying
```
Comment (spouliot) [inline src/foundation.cs]: should be `NSCopying` not `INSCopying`

**Comment by chamons** [comment]
@spouliot does that mean that this line already in the file is wrong as well?

	public interface NSDateComponents : NSSecureCoding, NSCopying, INSCopying, INSSecureCoding, INativeObject {

Should it just be NSCopying with no INSCopying?

**Comment by chamons** [comment]
Looks like they route to a concrete class:

002eb97c t -[_NSConcreteUserNotification additionalActions]

Teaching test.

**Comment by spouliot** [comment]
@chamons half wrong, the `INS*` does **nothing** in our *platform* bindings - we need the interface in the file so it can compile (when used in signatures) but it;s just noise on class declaration. What's really wrong is using `INS*` without the `NS*`.

That's different for user bindings since they are not implementing the protocol but must refer to the interface.

**Pull request merged**